pkg/proxy/iptables/proxier.go

```diff
@@ -1517,8 +1524,10 @@ func (proxier *Proxier) syncProxyRules() {
 		"numEndpoints", totalEndpoints,
 		"numFilterChains", proxier.filterChains.Lines(),
 		"numFilterRules", proxier.filterRules.Lines(),
-		"numNATChains", proxier.natChains.Lines(),
-		"numNATRules", proxier.natRules.Lines(),
+		"totalNATChains", proxier.natChains.Lines()+skippedNatChains.Lines(),
+		"totalNATRules", proxier.natRules.Lines()+skippedNatRules.Lines()-deletedChains,
```

Comment:
a comment explaining this maths can be useful 😄

Reply:
good call, since I was computing `totalNATChains` wrong...